herbert_core/applications/mfit_funcs/slow_func1d.m

```diff
@@ -0,0 +1,44 @@
+function y = slow_func1d (x, p, funchandle, nslow)
```

Comment:
Rather than defining this function multiple times, an alternative would be to pass `x` in as a cell array and use cellarray expansion to fill the funchandle.